### ISSUE TITLE
20.0.1 errors

### ISSUE THREAD

**Issue by koehn** [revpriest/listman#7]
I have installed Listman 20.0.1 on Nextcloud 22.0.0 (docker image) and configured the SMTP server settings. 

When I try to send an email, these errors show up in the log:

```
[PHP] Error: Undefined variable: plain at /var/www/html/custom_apps/listman/lib/Service/ListmanService.php#127

GET /index.phpman/message-view/[redacted]
from [redacted] by [redacted] at 2021-07-15T15:30:21+02:00
```

and also:
```
[PHP] Error: Array to string conversion at /var/www/html/lib/private/Security/CSP/ContentSecurityPolicyManager.php#83

GET /index.phpman/message-view/[redacted]
from [redacted] by [redacted] at 2021-07-15T15:30:21+02:00
```

and finally:
```
[PHP] Error: array_flip(): Can only flip STRING and INTEGER values! at /var/www/html/lib/public/AppFramework/Http/EmptyContentSecurityPolicy.php#442

GET /index.phpman/message-view/[redacted]
from [redacted] by [redacted] at 2021-07-15T15:30:21+02:00
```

**Comment by revpriest** [comment]
Thanks. I can see at least one problem there. Isn't failing on my machine but probably different PHP versions or different policies in 'em.

Will try and see what can be done about it this weekend.

**Comment by koehn** [comment]
Thanks! Looks like a great tool. I just tried using the subscribe form, and also got these errors:

```
[PHP] Error: Undefined variable: defaut at /var/www/html/custom_apps/listman/lib/Db/SettingsMapper.php#44

POST /index.phpman/subscribe/[redacted]
from [redacted] at 2021-07-15T15:45:42+02:00
```
```[PHP] Error: require(/var/www/html/custom_apps/listman/lib/Service/../../vendor/autoload.php): failed to open stream: No such file or directory at /var/www/html/custom_apps/listman/lib/Service/ListmanService.php#777

POST /index.phpman/subscribe/[redacted]
from [redacted] at 2021-07-15T15:45:42+02:00
```
```
[PHP] Error: require(): Failed opening required '/var/www/html/custom_apps/listman/lib/Service/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps') at /var/www/html/custom_apps/listman/lib/Service/ListmanService.php#777

POST /index.phpman/subscribe/[redacted]
from [redacted] at 2021-07-15T15:45:42+02:00
```

**Comment by revpriest** [comment]
I fixed the one problem where the variable wasn't defined until used. I dunno why my php let me get away with that. If I've turned on a slack-on-variables php setting on my setup I can't find it.

I fixed the other one which was a misspelling in an exception I'd presumably never encountered. I think just for settings that don't exist yet.

Both should be fixed in the new version 20.0.2 published just now.

I don't really understand what the others are, and assume they're just a cascade of things going wrong after those two things went wrong and hopefully won't happen if those two things don't go wrong.

:crossed_fingers: 

Let me know if that doesn't fix it.
